src/terminal.ts

```
@@ -28,6 +36,10 @@ export abstract class Terminal implements ITerminal {
   protected _writable: boolean;
 
   protected _internalee: EventEmitter;
+  protected _writeMethod: (data: string) => void = () => {};
```

I don't think we need to assign this yet as strict null checks isn't on atm?

---

Well technically now its possible that a derived class just forgets to assign its own impl to this - `Terminal.write` would crash (or need another nonsense check `if this._writeMethod`).

---

An NPE would be better than a silent failure though, I can look into strict issues when https://github.com/microsoft/node-pty/issues/306 is tackled

---

Ok.